src/System Application/App/AI/src/Copilot/CopilotCapabilityImpl.Codeunit.al

```diff
@@ -277,6 +277,10 @@ codeunit 7774 "Copilot Capability Impl"
     begin
         CopilotCapability := Enum::"Copilot Capability".FromInteger(Capability);
 
+        if (CopilotCapability <> Enum::"Copilot Capability"::Chat) or
```

Comment:
Not sure if we should throw an error here.

Reply:
Feel free to  resolve it if there is reason we cannot throw error from here

Reply:
We should not be throwing an error. The subscriber should not handle this if it is not of these types